pkgs/by-name/ep/epic5/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  ruby,
+  fetchurl,
+  openssl,
+  ncurses,
+  libiconv,
+  tcl,
+  coreutils,
+  fetchpatch,
+  libxcrypt,
+  perl,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "epic5";
+  version = "3.0";
+
+  src = fetchurl {
+    url = "http://ftp.epicsol.org/pub/epic/EPIC5-PRODUCTION/epic5-${finalAttrs.version}.tar.xz";
+    hash = "sha256-ltRzUME6PZkBnaDmoEsMf4Datt26WQvMZ527iswXeaE=";
+  };
+
+  # Darwin needs libiconv, tcl; while Linux build don't
```

Comment:
```suggestion
```
that comment just describes the code with no extra information, so we don't need it